### PR TITLE
Fix import of inflection library

### PR DESCRIPTION
**Pull request by McGiverGim**
This PR https://github.com/betaflight/betaflight-configurator/pull/3030 breaks the `inflection` library. In this new version the library has moved to a `node.js` library, with `require` to use it, that we don't use. This PR moves the path to point to the new location of the library.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
